### PR TITLE
Add visibility extension layer

### LINKED ISSUE #104: Visibility classes extension
Queryset needs to be filtered per node type to restrict visibility. For this a new configuration option (extension point) is added named `VISIBILITY_CLASSES` where a list of visibility classes can be configured. Those classes will be chained.

Following classes would be implemented to start with:
`BaseVisibility` see below for details
`Authenticated` only authenticated users may see any nodes, otherwise none.
`All` all items are visible (default)

`BaseVisibility` allows overwriting of `get_queryset` for any node type by naming convention. e.g. for node form it could be overwritten with method `get_queryset_for_form`.

### PR DESCRIPTION
**Pull request by sliverc**
Fixes #104 

Documentation will be addressed with #103

### CODE REVIEW

File: caluma/conftest.py [hunk 1]
```diff
@@ -37,3 +39,41 @@ def assert_execution_result(self, result):
 
     with GraphQlSnapshotTest(request) as snapshot_test:
         yield snapshot_test
+
+
+@pytest.fixture
+def info(rf):
```
Comment (winged) [inline caluma/conftest.py]: What is this info fixture? Might need a docstring

File: caluma/settings.py [hunk 2]
```diff
@@ -125,3 +126,12 @@ def parse_admins(admins):  # pragma: no cover
 OIDC_VALIDATE_CLAIMS_OPTIONS = env.dict(
     "OIDC_VALIDATE_CLAIMS_OPTIONS", cast={"value": bool}, default=None
 )
+
+# Extensions
+
+VISIBILITY_CLASSES = [
+    import_string(cls)()
```
Comment (winged) [inline caluma/settings.py]: In my understanding, when you call `import_string(cls)()` like this, you'll get an instance, not a class. So either the setting's name is wrong or you should only fetch the classes

File: caluma/conftest.py (hunk 1 above)
Comment (sliverc) [reply caluma/conftest.py]: Added docstring.

File: caluma/settings.py (hunk 2 above)
Comment (sliverc) [reply caluma/settings.py]: fair point. changed it so there are actual classes here.

**Pull request merged**